customized_list_view/tests/test_customized_list_view.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2024 ooops404
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import common
+
+
+class TestCustomizedListView(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestCustomizedListView, cls).setUpClass()
+        cls.view_model = cls.env["ir.ui.view"].sudo()
+        cls.users_model = cls.env["ir.model"]._get("res.users")
+        cls.view_users_tree = cls.env.ref("base.view_users_tree")
+
+    def test_all_customized_list_view(self):
```

Comment:
Could you add docstring where described what you do in this test. 
Please add message for each assert case.